core/templates/pages/exploration-editor-page/statistics-tab/charts/pie-chart.component.spec.ts

```diff
@@ -31,14 +31,10 @@ describe('Pie Chart component', function() {
       getResizeEvent: () => of(resizeEvent)
```

Comment:
Extremely painful code to maintain. Did my best to reduce cruft while having something that compiles/passes.